docs/components/LogoContext/index.tsx

```diff
@@ -0,0 +1,169 @@
+import { useEffect, useCallback, useState, useRef } from "react";
+import { useTheme } from "nextra-theme-docs";
+import Link from "next/link";
+import classNames from "classnames";
+import { VercelLogo } from "./icons";
+import { PRODUCT_MENU_ITEMS, PLATFORM_MENU_ITEMS } from "./items";
+import type { MenuItemProps } from "./types";
+import { MouseEvent } from "react";
+import { useTurboSite } from "../SiteSwitcher";
+
+function MenuDivider({ children, ...other }: { children: string }) {
+  return (
+    <div
```

Comment:
This should probably be a `h3` so a11y tools can use it as a landmark to jump between the sections.